package.json

```diff
@@ -114,9 +113,8 @@
   },
   "peerDependencies": {
     "moment": "^2.18.1",
-    "react": ">=0.14",
-    "react-dom": ">=0.14",
-    "react-addons-shallow-compare": ">=0.14"
+    "react": ">=15.3.0",
+    "react-dom": ">=15.3.0"
```

Comment:
@ingoclaro if preact doesn't support shouldComponentUpdate, that sounds like a bug in preact :-)

@majapw enzyme supports 0.13 still. I think that a) a general guideline should be "latest, + 2 versions back" (we never supported 0.13 here, ofc, but we probably should have) and b) we shouldn't drop support for something without a good reason.

PureComponent in React iirc is just a shouldComponentUpdate that's written for you - imo that's not a good enough reason by itself.